kube-controllers/pkg/controllers/node/ipam.go

```diff
@@ -123,11 +121,6 @@ func NewIPAMController(cfg config.NodeControllerConfig, c client.Interface, cs k
 		client:    c,
 		clientset: cs,
 		config:    cfg,
-		rl: workqueue.NewTypedMaxOfRateLimiter(
```

Comment:
tbh I'm not really sure why we had this rate limiting in place to begin with. I think removing it is OK to do? It was artificially slowing down our ability to release IPAM block affinities.

Reply:
the "This is only for retry speed" makes me think this is to implement some kind of exponential backoff for retries? should be useful to keep if so, but not sure if that's the only thing this did...

Reply:
Yeah, but I think it was an outdated comment or just wrong. The ratelimited would block cleanup of other nodes if a single node hit an issue, and didn't seem to be related to retries.

Plus, the second argument here applies to every request - not just failed ones - which enforced an intrinsic 10 per second limit.